inference-engine/src/mkldnn_plugin/mkldnn_extension.cpp

```diff
@@ -0,0 +1,204 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "mkldnn_extension.h"
+#include "ngraph_transformations/op/fully_connected.hpp"
+#include "ngraph_transformations/op/leaky_relu.hpp"
+#include "ngraph_transformations/op/power_static.hpp"
+#include "ngraph_transformations/op/swish_cpu.hpp"
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph_ops/type_relaxed.hpp>
+
+#include <mutex>
+
+namespace MKLDNNPlugin {
+
+void MKLDNNExtension::GetVersion(const InferenceEngine::Version*& versionInfo) const noexcept {
+    static const InferenceEngine::Version version = {
+        {1, 0},             // extension API version
+        "1.0",
+        "MKLDNNExtension"   // extension description message
+    };
+
+    versionInfo = &version;
+}
+
+void MKLDNNExtension::Unload() noexcept {}
+
+std::map<std::string, ngraph::OpSet> MKLDNNExtension::getOpSets() {
+    auto cpu_plugin_opset = []() {
+        ngraph::OpSet opset;
+
+#define NGRAPH_OP(NAME, NAMESPACE) opset.insert<NAMESPACE::NAME>();
+        NGRAPH_OP(FullyConnectedNode, MKLDNNPlugin)
+        NGRAPH_OP(LeakyReluNode, MKLDNNPlugin)
+        NGRAPH_OP(PowerStaticNode, MKLDNNPlugin)
+        NGRAPH_OP(SwishNode, MKLDNNPlugin)
+#undef NGRAPH_OP
+
+        return opset;
+    };
+
+    auto type_relaxed_opset = []() {
+        ngraph::OpSet opset;
+
+#define NGRAPH_OP(NAME, NAMESPACE) opset.insert<ngraph::op::TypeRelaxed<NAMESPACE::NAME>>();
+//        NGRAPH_OP(FullyConnectedNode, MKLDNNPlugin)
```

Comment:
Why do we have a lot of commented operations?

Reply:
I think it will be more convenient if we decide to add a new layer to type relaxed opset. We just need to uncomment the required layer.

Reply:
All comments removed